.eslintrc.js

```diff
@@ -74,5 +74,21 @@ module.exports = {
     __TESTING__: true,
     _: false,
     AppConfig: true
-  }
+  },
+  overrides: [{
+    // ES 2018 - specify migrated files and folders here
+    files: [
+      'testutils/data.server.testutils.js',
+      'modules/references/server/**',
+      'modules/references/tests/server/**'
```

Comment:
Nice and simple 👌

Reply:
Yes, @nicksellen 's idea. :bulb: I had additional `.eslintrc` all around the place. :)